packages/react-examples/src/react-experiments/SelectedPeopleList/SelectedPeopleList.WithEditInContextMenu.Example.tsx

```diff
@@ -136,8 +123,17 @@ export class SelectedPeopleListWithEditInContextMenuExample extends React.Compon
     });
 
     return copyText;
-  }
-}
+  };
+
+  return (
+    <div className={'ms-BasePicker-text'}>
+      Right click any persona to open the context menu
+      <br />
+      <PrimaryButton text="Add another item" onClick={_onAddItemButtonClicked} />
+      {_renderExtendedPicker()}
```

Comment:
you can just put the contents of _renderExtenedPicker here if you want..
also if you decide not to, then we may want to rename ExtendedPicker, since it is not :)